src/utils/Utils.ts

```diff
@@ -0,0 +1,49 @@
+import { ResourceType, ResourceTypeAttribute } from '@/apis/rembrandt/rembrandt';
+import { ListElement } from '@/components/ListSection.vue';
+
+export default class Utils {
+  // region public static methods
+  public static resourceTypesToList(resourceTypes: ResourceType[]): ListElement[] {
+    return resourceTypes.map((resourceType) => {
+      return {
+        id: resourceType.id || resourceType.name,
+        firstValue: resourceType.name,
+        secondValue: resourceType.parentType ? `Parent Type: ${resourceType.parentType}` : '',
+        thirdValue: resourceType.abstract ? 'Abstract Type' : `300 Instances`,
```

Comment:
We should not forget this hard-coded value!
Maybe we can offer an extended api route offering statistics on elements (where a call to resourceType could include the number of instances, number of subtypes, and so on).

Reply:
A route for statistics sounds nice :)

Reply:
I will create a new issue for that in the back end :)